NBISweden/sda-cli
# Feature/sda download

**Related issue(s) and PR(s)**  
This PR closes #372.


**Description**
A new feature in the sda-cli is implemented to download files from the download service endpoint by giving the dataset stable_id and the inbox path of the file without the uploader's user id

**How to test**
By testing that just run the setup script from the root folder: `bash .github/integration/setup/setup.sh`
and when it is done one can download the file by running: `./sda-cli sda-download -config testing/s3cmd-download.conf -dataset https://doi.example/ty009.sfrrss/600.45asasga -url http://localhost:8080 -outdir test-download main/subfolder/dummy_data.c4gh`
Then check if the file `test-download/main/subfolder/dummy_data` exists and if the result by running `head test-download/main/subfolder/dummy_data` is "THIS FILE IS JUST DUMMY DATA"
